here-naksha-app-service/src/main/java/com/here/naksha/app/service/http/ops/PropertySelectionUtil.java

```diff
@@ -66,6 +68,9 @@ private PropertySelectionUtil() {}
    *    id
    *    type
    *    geometry
+   *
+   * Note - Entire 'geometry' object is added by default, ONLY if it is not already (partially) requested.
```

Comment:
Personally this would be easier for me to understand: Entire 'geometry' object is added by default, UNLESS it is (partially) requested.

Maybe the partial word can be emphasized with capital letters as well, depending on how important it is.

Reply:
this sounds better. corrected.